### PR TITLE
chore: Upgrade versions

### PR DESCRIPTION
**Pull request by yammesicka**

### CODE REVIEW

File: lms/lmstests/public/linters/tasks.py [hunk 1]
```diff
@@ -12,10 +14,15 @@
...
-        _logger.info('Start running check solution %s', solution_pk)
-        checker = services.LinterChecker(solution_pk, _logger)
         checker.initialize()
+    except models.Solution.DoesNotExist:
+        _logger.exception('Solution %s does not exist', solution_pk)
+
+    try:
         checker.run_check()
-    except Exception:
+    except (CeleryError.TaskError, OperationalError):
```
Comment (orronai) [inline lms/lmstests/public/linters/tasks.py:27]: Are you sure it's not just celery.exceptions.TaskError?
https://docs.celeryproject.org/en/stable/reference/celery.exceptions.html#celery.exceptions.TaskError

Comment (yammesicka) [reply lms/lmstests/public/linters/tasks.py:27]: 'cause `OperationalError` is not under `TaskError`
It basically should catch connection errors

Comment (yammesicka) [reply lms/lmstests/public/linters/tasks.py:27]: Now I got what you meant XD Yeah, you're right, fixing it. Great catch

File: tests/conftest.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import logging
 import os
 import random
 import string
 from typing import List, Optional
 
+from _pytest.logging import caplog as _caplog  # NOQA: F401
 from flask import template_rendered
 from flask.testing import FlaskClient
+from loguru import logger
```
Comment (orronai) [inline tests/conftest.py:12]: Why not using our logger called `log` from `lms.utils.log`?

**Pull request merged**